packages/kit/src/exports/vite/dev/index.js

```diff
@@ -197,19 +200,12 @@ export async function dev(vite, vite_config, svelte_config) {
 							const styles = {};
 
 							for (const dep of deps) {
-								const url = new URL(dep.url, 'dummy:/');
-								const query = url.searchParams;
-
-								if (
-									(isCSSRequest(dep.file) ||
-										(query.has('svelte') && query.get('type') === 'style')) &&
```

Comment:
I removed this specific handling in the new code since `isCSSRequest` should already be capturing any instances of this. I tested in the playground locally and the inlined-styles to prevent fouc still worked.